sdk/synapse/synapse-spark/test/utils/utils.common.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { env } from "@azure/test-utils-recorder";
+import * as assert from "assert";
+
+// Async iterator's polyfill for Node 8
```

Comment:
does this really need to be inlined? `core-asynciterator-polyfill` is supposed to take care of this
 
https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/core/core-asynciterator-polyfill

Reply:
Will remove it.

---
In reply to: [521615917](https://github.com/Azure/azure-sdk-for-js/pull/12487#discussion_r521615917) [](ancestors = 521615917)